scripts/install-dotnet.sh

```diff
@@ -100,6 +100,10 @@ get_os_download_name_from_platform() {
             echo "ubuntu.16.10"
             return 0
             ;;
+        "ubuntu.18.04")
+            echo "ubuntu"
```

Comment:
We should get rid of the .NET Core 1.0 installer dependency and change this to return ubuntu18.04.  As a super short term hack we could probably return 16.04 for now with a // bugbug comment, until the 1.0 -> 2.1 change is in